### PR TITLE
Feature/header scss

### PR DESCRIPTION
**Pull request by vd1992**
Pull

### CODE REVIEW

File: src/sass/header.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    color:$third-color;
+    display: flex;
+    align-items: center;
+    
+    .header-logo{
+        padding-left:25px;
+        margin-right:auto;
+        color:$third-color;
+    }
+    .header-logo i{
```
Comment (akmadsen) [inline src/sass/header.scss]: - [ ] Nest the i inside the header-logo.

File: src/sass/header.scss [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        color:$third-color
+    }
+    
+    .header-link {
+        height:100%;
+        padding-top:27px;
+        padding-right:50px;
+    }
+
+    .header-link a{
```
Comment (akmadsen) [inline src/sass/header.scss]: - [ ] nest the `a` inside the `header-link`

File: src/sass/header.scss [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        height:100%;
+        text-decoration: none;
+        color:$third-color;
+    }
+
+    .header-link a:hover{
```
Comment (akmadsen) [inline src/sass/header.scss]: - [ ] nest the `a:hover` inside the `header-link` class

**Comment by vd1992** [comment]
@akmadsen 
Updated nesting, please look over again.

**Pull request merged**